pkg/apis/networking/validation/validation.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/google/go-cmp/cmp"
```

Comment:
IIRC, go-cmp isn't recommended for production code, only for testing. I think it has a tendency to panic if it sees something it doesn't like.

Reply:
Removed go-cmp, using apivalidation.ValidateImmutableField to validate that the fields are not mutated